README.md

```diff
@@ -1,2 +1,22 @@
 # task-launcher-app
+
 Landing page with links/buttons to go to tasks
+
+## Getting Started
```

Comment:
throw a note on here about where the config lives if anyone wants to use it for a different study.  @mirestrepo should the title of the landing page be butler specific?

Reply:
@mcmcgrath13 is the config the array for the tasks? Also related to that, should I put that as a separate json object? right now, I just have it inside App.js. If it should be a separate file, where should that file be located?

Reply:
I think the current level of abstraction is fine for what this project is.  Just note that if someone wants to change the tasks, what updates they need to make and where (e.g. update this variable, add the svg file here).